### ISSUE TITLE
Is there no way to simply restore a file?

### ISSUE THREAD

**Issue by QuarkZ26** [garethgeorge/backrest#436]
Hello,

After backing up, when using the Restore to path option, I have to not only select an empty path, but it restores the whole path with subfolders.

Let's say I want to restore /home/user/Docs/stuff.pdf into /home/user/Docs, it's not possible, I would have to use a subfolder, and that subfolder will have the whole /subfolder/home/user/Docs/stuff.pdf folders, which doesn't really make much sense to me. I should be able to restore simply the file, to any location, and overwrite if needed.

Secondly, if you use root to backup, and restore the same way, /subfolder will be owned by root, which makes it quite annoying.

Is it possible to simply restore the file itself in Docs, with proper permissions or will have have to use Restic directly for that?

Thanks.

**Comment by garethgeorge** [comment]
Good point, looks like Backrest could be using the alternative restore syntax e.g. `restic restore <snapshot ID>:<path>` to avoid creating intermediate directories. I'll look at switching to switching over in the next minor version release.

**Comment by garethgeorge** [comment]
Implemented some of these changes in https://github.com/garethgeorge/backrest/commit/af09e47cdda921eb11cab970939740adb1612af4 , skipping windows for now.

**Comment by QuarkZ26** [comment]
Even after update 1.5.0, it's still not restoring the way it should. Restoring a file still complains about the folder not being empty, and will only restore into a subfolder, which isn't ideal. Not only, but the subfolder is owned by root.
Example, restoring file.pdf into Docs, will create file.pdf/file.pdf.

Any file or folder should be restoreable into whatever root folder chosen (Docs/file.pdf) and be owned by the user, and it should overwrite any file in the folder.

**Comment by Vyerni** [comment]
Any news on this?

I'm testing a full disaster restore, and it's kind of painful with things not restoring in place

**Comment by britshickhouse** [comment]
Yes please! My wishlist would be:

1. Restore in place (overwrite folder if need be)
2. Restore original permissions

Duplicati on Linux/Docker is able to do these, though I'd rather move to Restic/Backrest as Duplicate has some reliability issues (not dealbreakers, just annoying)

**Comment by davidr521** [comment]
@garethgeorge Any update/fix for this?
